pkg/generate/app/environmentfile.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"bufio"
+	"os"
+)
+
+// IsPossibleEnvironmentFile returns true if the argument is a file. I am not
+// entirely sure, why we are using this function, as this is just a wrapper,
+// we are not doing any additional validations here. This is something that I
+// am borrowing from templates file. Need feedback on this.
+func IsPossibleEnvironmentFile(value string) bool {
+	var fileexists bool = true
+	_, err := os.Stat(value)
+	if err != nil {
+		fileexists = false
+	}
+	return isFile(value) && fileexists
+}
+
+// getEnvironmentVars reads a whole file into memory and returns string
+// array. This could be a problem if we are reading a very large file. I am
+// not putting any validation for file, if it contains a key=value pair as
+// that should be handled in environment variables that we are passing.
+func GetEnvironmentVars(path string) []string {
```

Comment:
> error: environment variables must be of the form key=value: oc new-app openshift/postgresql-92-centos7 --env-file=envfile 

is that actually part of the output?  doesn't look right.

otherwise i guess it's ok.